guides/common/modules/proc_registering-hosts-for-conversion.adoc

```diff
@@ -9,7 +9,6 @@ Activation keys produced by the Ansible role are as follows:
 * convert2rhel_centos8 - used for registering centos8 hosts.
 * convert2rhel_oracle7 - used for registering oracle7 hosts.
 * convert2rhel_oracle8 - used for registering oracle8 hosts.
-* convert2rhel_rhel7 - used for re-registering a host converted to RHEL 7.
```

Comment:
I'm not sure this is correct. This procedure is to convert hosts, not the Satellite server. RHEL 7 hosts will be supported for a while.